src/js/big-play-button.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @file Needed for full path retrieval
```

Comment:
Saying this each time isn't ideal. Can you explain why it's needed, and should we try to put real file descriptions for each of these? It could be event as generic as the file name.